tensorflow_addons/optimizers/cyclical_learning_rate.py

```diff
@@ -169,6 +170,11 @@ def __init__(
             name=name,
         )
 
+    def get_config(self):
+        super_config = super().get_config()
+        super_config = {k: v for k, v in super_config.items() if k != "scale_fn"}
+        return {**super_config}
```

Comment:
Would you mind making these serializations explicit for each subclass. It's a bit brittle to depend on that check going forward.

Reply:
Done.